frame/beefy-mmr/src/mock.rs

```diff
@@ -57,11 +59,6 @@ construct_runtime!(
 	}
 );
 
-parameter_types! {
```

Comment:
There are also other `parameter_types` that you can replace in this file, such as:
```
parameter_types! {
	pub const Period: u64 = 1;
	pub const Offset: u64 = 0;
}
```

Reply:
Same for  `frame/beefy/src/mock.rs`

Reply:
I have refractored this now @acatangiu